lib/scarpe/wv/motion.rb

```diff
@@ -0,0 +1,101 @@
+# frozen_string_literal: true
+
+class Scarpe
+    class WebviewMotion < Scarpe::WebviewWidget
+      def initialize(properties)
+        super(properties)
+  
+        bind("mousemove") do |client_x, client_y|
+          send_display_event(event_name: "mousemove", target: shoes_linkable_id, client_x: client_x, client_y: client_y)
+        end
+  
+        bind("mousedown") do |client_x, client_y|
+          send_display_event(event_name: "mousedown", target: shoes_linkable_id, client_x: client_x, client_y: client_y)
+        end
+  
+        bind("mouseup") do
+          send_display_event(event_name: "mouseup", target: shoes_linkable_id)
+        end
+  
+        bind("mouseleave") do
+          send_display_event(event_name: "mouseleave", target: shoes_linkable_id)
+        end
+      end
+  
+      def element(&block)
+        onmousemove = handler_js_code("mousemove", "event.clientX", "event.clientY")
+        onmousedown = handler_js_code("mousedown", "event.clientX", "event.clientY")
+        onmouseup = handler_js_code("mouseup")
+        onmouseleave = handler_js_code("mouseleave")
+  
+        HTML.render do |h|
+          h.div(id: html_id, style: style) do
+            h.canvas(
+              width: "400",
+              height: "500",
+              viewBox: "0 0 400 500",
+              style: "background-color: gray",
+              onmousemove: onmousemove,
+              onmousedown: onmousedown,
+              onmouseup: onmouseup,
+              onmouseleave: onmouseleave,
+              oncreate: handler_js_code(draw_script),
```

Comment:
hmm can someone give me some pointers on what im dng wrng here.

Reply:
This looks reasonable at first glance. What example are you using?

Reply:
as of now just

```
Shoes.app do
motion
end
```

(its not crct usage acc to manual.md but if it works changing it acc to our usage is not the issue)

Reply:
Ah! This is the only widget with no display properties. If you add "display_properties :dummy" it works fine. I'll add a fix in another PR so that you don't need to do that.

(I don't know if motion should have display properties later, but I should fix it so it works with none.)

Reply:
hm the issue i am having is not that(not sure y i didnt got tht error or maybe i removed while commitin?)
but
 its not drawing anythin on screen for me
 while its recording coordinates n all

Reply:
Yup, that's the error I mean. Here's the PR to fix it: https://github.com/scarpe-team/scarpe/pull/211